pilot/helpers/agents/Developer.py

```diff
@@ -769,7 +774,7 @@ def continue_development(self, iteration_convo, last_branch_name, continue_descr
                     "files": self.project.get_all_coded_files(),
                     "user_feedback": user_feedback,
                     "user_feedback_qa": user_feedback_qa,
-                    "previous_solutions": llm_solutions[-3:],
+                    "previous_solutions": llm_solutions,
```

Comment:
Instead of getting last 3 here, I take last 3 inside `get_alternative_solutions.prompt`. Reason for that is that we are storing in db all data in column `prompt_data` and this way we will not store only last 3 but all previous llm_solutions so that i can access all of them when updating dev plan with `update_plan.prompt`